### ISSUE TITLE
Improper Names

### ISSUE THREAD

**Issue by azder** [WowUp/WowUp#527]
Top left of the window contains a drop down list for switching with names `Retail`, `Classic`, `Retail PTR`, `Beta`. 

There isn't a game named Retail, only World of Warcraft. One only needs to see the Battle.Net launcher options above the big Play button

![github-bug-wow-99294781-d0267300-2844-11eb-8cfe-956e864dc5ce](https://user-images.githubusercontent.com/61147/100132803-3f741680-2e86-11eb-9ec8-086410558318.png)

One might go with the offical names (or abbreviations like WoW, WoW PTR, Beta, PTR Classic) or with just the directory names which are unambiguous and brief

![github-bug-wow-99295258-7d998680-2845-11eb-8125-f041706a3773](https://user-images.githubusercontent.com/61147/100132849-4ac74200-2e86-11eb-9dc6-77a412ed5c48.png)

**Comment by linaori** [comment]
It's called "Retail" because that's the specific client. "WoW Client" is ambiguous, because the question you get back is "which wow client?".

Is there a particular issue that needs solving? I would rather see it named "Retail" because that's something people usually know as this term is used pretty much everywhere.

**Weak Aura Companion App calls it "Retail":**
![image](https://user-images.githubusercontent.com/1754678/100133980-c37ace00-2e87-11eb-9c64-66bbc511b2b9.png)

---

**Ajour calls it "Retail":**
![image](https://user-images.githubusercontent.com/1754678/100134189-1e142a00-2e88-11eb-9683-9ac9529efb2a.png)
(source: https://www.getajour.com/)

---

**Twitch lets you select the `_retail_` directory:**
![image](https://user-images.githubusercontent.com/1754678/100134309-4ef45f00-2e88-11eb-87b5-3be467f5f400.png)
(source: https://www.youtube.com/watch?v=acyYsJJX2Fs)

---

**CurseBreaker refers to it as "Retail":**
(source: https://github.com/AcidWeb/CurseBreaker)

---

**Strongbox lets you select the `_retail_` directory:**
![image](https://user-images.githubusercontent.com/1754678/100134648-cf1ac480-2e88-11eb-88c1-37059140ca99.png)
(source: https://github.com/ogri-la/strongbox)

Ogri-la refers to it as "Retail" as well on https://ogri-la.github.io/wow-addon-managers/

---

**Singularity refers to it as "Retail":**
![image](https://user-images.githubusercontent.com/1754678/100134748-f96c8200-2e88-11eb-9cea-f4dba9590070.png)
(source: https://singularitymods.com/)

---

**Overwolf refers to it as "Retail"**
![image](https://user-images.githubusercontent.com/1754678/100134955-40f30e00-2e89-11eb-8fbf-247c57a365a1.png)
(source: https://www.forbes.com/sites/hnewman/2020/11/06/overwolf-changes-privacy-features-and-ads-in-curseforge-app-for-minecraft-and-wow/)

---

**Wowhead calls it "Live", I suspect because they always had Live vs Beta vs PTR:**
![image](https://user-images.githubusercontent.com/1754678/100134087-f4f39980-2e87-11eb-9b46-b774a3874eb9.png)

**Comment by azder** [comment]
TL;DR:
> Is there a particular issue that needs solving?
Naming consistency.

Directory names can't clash, Battle.net client has the offical names.
Both are unambiguous and from the actual Creator, thus authorative, not 3rd party sources.


The longer part:

In a world of many third party software provided, it'd be nice if 3rd parties just followed one or the other from Blizzard, and not re-invent monikers.

Battle.net client already has **unambiguous** names, I also suggested going by **directory names** (one of which is named retail, so it is not an issue of using the term in this context).

On the term, there isn't a game named Retail, there is a directory named retail.
- On the Blizzard sales side, retail is the game time payed, thus, both WoW and WoW Classic can be played, thus both are Retail.
- On the Blizzard dev side, naming is hard, they botched it, we're stuck with it, let's just use it **exactly** as it is.

Arguments of the kind: Everyone calls the Earth flat so I will call the it flat as well isn't a good one.
Not the least because some say "flat" other "flaten", yet others "flat ptr" etc.



P.S. on the Ajour argument https://github.com/casperstorm/ajour/issues/359

I don't mind these issues not being resolved, I was just curious how you'd deal with it.

**Comment by linaori** [comment]
> Naming consistency.

Which is why it's called "Retail", because this is what people are used to with the tooling.

> Arguments of the kind: Everyone calls the Earth flat so I will call the it flat as well isn't a good one.

See above, naming consistency.

> Battle.net client already has unambiguous names

As I've told you in my previous example, the names _are_ ambiguous. The retail client is "World of Warcraft", and when you talk about a client that's named the same as the game, you can mean any client version. Retail is a good distinction, and I've seen 0 issues come by because of the name. By now the majority understands what client retail is.

> I don't mind these issues not being resolved, I was just curious how you'd deal with it.

You're the first person to come up with this and we don't see the name as a problem. If there are more people that are really confused by it and it's causing us to have to give more support because of it, we can look into changing it. For now I think it's good that the majority of tooling is using the same naming conventions so that people know what we're talking about.

**Comment by azder** [comment]
**TL;DR**: it is simple to follow directory naming convention without adding or substracting.

You're still stuck at "this is why we use the word retail" argument, when I say:

> Top left of the window contains a drop down list for switching with names Retail, Classic, **Retail PTR**, Beta.

> Directory names can't clash

>  let's just use it exactly as it is (e.g. no `_retail_ptr_` directory).

The majority of "tooling" is diverging because of improvisations and focusing too much to responding one critisism, ignoring other. Yes, retail is wrong for product name (they're called live), yet, you can't seem to go past that and see 

**retail is OK for directory name.**

Out of the two proposed consistencies, you don't seem to have a problem of improvising a third one, eventually naming a directory _ptr_ as Retail PTR client. Other tools also don't have a problem of improvising, but the issue is, some of them have oversaturated the UI because of improvisation.

I shall put this to rest now, nothing more to add as I'm sure you have more pressing maters to resolve.
Best of luck and keep up the good work.
